avocado/core/job.py

```diff
@@ -371,31 +370,15 @@ def _log_avocado_datadir():
         job_log.info('logs     ' + data_dir.get_logs_dir())
         job_log.info('')
 
-    def _log_variants_tree(self, variant):
```

Comment:
Do you remember this: https://github.com/avocado-framework/avocado/pull/1774#discussion_r100579547 compare to this PR it was quite simple and still it was hard to just squash two patches. Now if I'm to not just squash two commits, but basically merge 2 renames, where the first one happened at the beginning and the second one after complete move of the files to a different location, that basically means I have to rework everything. And I'm not talking about minor clashes, that in deed means developing it all over again as all the code will be incompatible, even though the result will be the same.

This is why I was asking whether it is really worth requiring this change. It will most probably be never seen by people as it'll be atomically merged and if we want to revert a part, we'd most probably revert everything. So do you see this change so important that it means compete rework of all the patches which are here, or do you think it's low impact enough to allow this namechange to be kept? (I'm not arguing about some of the other issues, but the namechange which brakes everything is something I'd like to avoid).